symfony.lock

```diff
@@ -303,9 +255,6 @@
     "psr/event-dispatcher": {
         "version": "1.0.0"
     },
-    "psr/http-client": {
```

Comment:
Looks like a lot of stuff was removed from the symfony.lock file. Can somebody explain why?

Reply:
it's lock file

Reply:
I know